SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialConditionMapFilter.h

```diff
@@ -31,7 +31,11 @@ class FSpatialConditionMapFilter
 #endif
 
 		// Build a ConditionMap. This code is taken directly from FRepLayout::RebuildConditionalProperties
+#if ENGINE_MINOR_VERSION <= 22
 		static_assert(COND_Max == 14, "We are expecting 14 rep conditions"); // Guard in case more are added.
+#else
+		static_assert(COND_Max == 16, "We are expecting 16 rep conditions"); // Guard in case more are added.
```

Comment:
2 conditions have been added in 4.23, but we've only added one more entry?

Reply:
This is a fun one. The old `COND_Max` used to be 14, they removed 14 and then made the new condition `COND_Never=15`. They then made the new `COND_Max=16. So basically they only added one new condition and they incremented twice :p